### PR TITLE
Updates to Gitlab PRs

### PR DESCRIPTION
**Pull request by AdeAttwood**
## JIRA

n/a

## Overview

**feat: use the default branch as the Gitlab PR target**

Previously, the target branch for the PR when using Gitlab was hardcoded
to "master". Now the repositories default branch will be used.

**refactor: the Gitlab PR body to be consistent with Github**

Now both Gitlab and Github have the same logic for creating the PR title
and body. It will use the first line of the commit as the title then
concatenate the commit body and the content of the PRBody file that is
passed in as a parameter

## Testing

Tested locally and have been using microplane with this patch for a while.

### CODE REVIEW

File: push/pushGitlab.go [hunk 1]
```diff
@@ -39,21 +39,24 @@ func GitlabPush(ctx context.Context, input Input, repoLimiter *time.Ticker, push
...
+	}
+
 	// Open a pull request, if one doesn't exist already
 	head := input.BranchName
-	base := "master"
+	base := project.DefaultBranch
 
 	// Determine MR title and body
 	// Title is first line of commit message.
 	// Body is given by body-file if it exists or is the remainder of the commit message after title.
```
[the remark this answers is not in the document]
Comment (nathanleiby) [reply push/pushGitlab.go:53]: Relatedly... given that the logic is now the same across Github and Gitlab, we could also refactor the logic into a small helper method that's re-used in both.

```go
title, body := getTitleBody(input Input)
```

**Review by nathanleiby — APPROVED** [review]
Thanks for the contribution! Looks good. 

I mentioned some optional nits if you have time to add them. Will merge as-is if you don't have time.

File: push/pushGitlab.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (AdeAttwood) [reply push/pushGitlab.go]: @nathanleiby Thanks for your comments. I have updated the second commit to include your suggested comments. I have also given it a quick test on GitLab. Sorry, I am unable to test on GitHub

**Review by nathanleiby — APPROVED** [review]
LGTM 🚢

**Comment by nathanleiby** [comment]
Released in v0.0.34 (https://github.com/Clever/microplane/commit/8e3e1dbf10a45ab00bb29236c01c1eda3086339f)

@AdeAttwood Thanks again for your contribution!

**Pull request merged**